### DISCUSSION TITLE
关于多个方案切换和中英文按键功能的想法

### DISCUSSION THREAD

**Discussion by DreamAfar** [imfuxiao/Hamster#66] [Ideas]
最近用了下cantaborad键盘，也是一个iOS端的开源Rime的输入法，但不支持自定义方案，发现他的一些功能交互做的很细致到位，值得借鉴。
https://github.com/Cantoboard/Cantoboard

它利用一个按键短按和长按切换不同的功能。
输入的编码，可以在中文英文之间来回切换而不被清掉。

展开多候选也可以各种过滤，不会中断输入的编码。

PS了个图，在仓输入法上的效果：
识别方案内的定义的全角半角标点、简繁切换切换等
![image](https://user-images.githubusercontent.com/69999980/232678838-1983d1ac-88c9-417d-92b1-bd6f931fbeb7.png)

![image](https://user-images.githubusercontent.com/69999980/232678857-9fb99cd8-b8df-4720-a08d-6ba7342acbcf.png)

录屏了个它的交互：
https://user-images.githubusercontent.com/69999980/232682079-fc17b131-e459-4b53-9a87-614937896bd2.mp4

**Comment by imfuxiao** [comment]
感谢反馈, 目前App确实有很多地方不完善. 

目前主要精力放在App的bug上. 您视频中的相关功能都在开发列表中, 后续都会有的.

**Reply by DreamAfar** [reply]
大佬，不急哈！我只是提个建议。
仓输入法目前已经很可以了，满足日常的输入。
那些功能可以后面慢慢实现。

**Reply by imfuxiao** [reply]
客气了, 需要你们的反馈, 只有大家不断提反馈意见, app才能越来越好用.
